### DISCUSSION TITLE
New to PBPK modeling

### DISCUSSION THREAD

**Discussion by pburkat** [Open-Systems-Pharmacology/Forum#784] [Questions & Problems] [answered]
Good morning, I am new to PBPK modeling, but have extensive experience with pharmacodynamic kinetic modeling. Are there any resources available that help in decision making for parameter modification to improve model fits to data?

**Comment by Christoph27** [comment]
Maybe the following tutorial is helpful: doi:10.1002/psp4.12134
Besides describing a typical overall strategy to build a PBPK model, also typically adjusted parameters are mentioned.

**Comment by pburkat** [comment]
How does one simulate drug concentration in compartments aside from peripheral venous blood and create a plot?

**Comment by Yuri05** [comment]
For **individual** simulations: s. [Running a simulation in an individual‌](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#running-a-simulation-in-an-individual) and [Analyzing results for a simulation in an individual‌](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#analyzing-results-for-a-simulation-in-an-individual)
For **population** simulations: s. [Running a population simulation‌](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#running-a-population-simulation) and [Analyzing a population simulation‌](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-simulations#analyzing-a-population-simulation)

**Comment by pburkat** [comment]
How does one increase simulation resolution in PKSim?

**Comment by Yuri05** [comment, marked as answer]
You can change the resolution here:
![grafik](https://user-images.githubusercontent.com/25061876/108698000-e018b780-7503-11eb-9977-1d959559f707.png)

**Comment by pburkat** [comment]
Where can I find information on how PKSIM handles compound transport into brain and all of its compartments?

**Comment by pburkat** [comment]
Are the differential equations in PK Sim permeability limited? Of which form?

**Comment by pburkat** [comment]
Please disregard my last 2 questions.

**Comment by StephanSchaller** [comment]
@pburkat , please open a new issue for each new question, otherwise your questions might be missed.

For the "permeability limited" question, please see https://github.com/Open-Systems-Pharmacology/Forum/issues/700
